lib/src/realm_class.dart

```diff
@@ -256,8 +255,10 @@ class _Scheduler {
     final sendPort = receivePort.sendPort;
     handle = realmCore.createScheduler(Isolate.current.hashCode, sendPort.nativePort);
 
-    //We use this to receive a SCHEDULER_FINALIZE_OR_PROCESS_EXIT notification on process exit to close the receivePort or the process with hang.
-    Isolate.spawn(_handler, 2, onExit: sendPort);
+    if (!isFlutterPlatform) {
```

Comment:
Are we sure it is safe when not running flutter?

Reply:
yes